src/Sylius/Behat/Page/Checkout/CheckoutPaymentStepInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Checkout;
+
+use Sylius\Behat\Page\ElementNotFoundException;
+use Sylius\Behat\Page\PageInterface;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+interface CheckoutPaymentStepInterface extends PageInterface
+{
+    /**
+     * @param string $paymentMethod
+     *
+     * @throws ElementNotFoundException
+     */
+    public function selectPaymentMethod($paymentMethod);
+
```

Comment:
Nope. Implementation of this function does not return anything nor taking any parameters.